tests/contrib/django/django_app/appsec_urls.py

```diff
@@ -11,6 +11,12 @@
 from ddtrace.appsec.trace_utils import block_request_if_user_blocked
 
 
+try:
+    from ddtrace.appsec.iast._ast.aspects import add_aspect
+except ImportError:
+    # Python 2 compatibility
+    from operator import add as add_aspect
```

Comment:
Do we need to run `ddtrace.appsec.iast._ast` tests with python 2? It seems like it is not supported

Reply:
We run this code with Python 2 for other ASM tests, but support for Python 2 isn't planned yet for IAST